### PR TITLE
Added parameters to Tag::renderTitle()

### PR DESCRIPTION
**Pull request by ekmst**
Hello!

* Type: new feature
* Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change: Changed the `Phalcon\Tag::renderTitle()` parameters such as `Phalcon\Tag::getTitle()`

Thanks

### CODE REVIEW

**Comment by niden** [comment]
@ekmst Would you be so kind as to include in this PR the same change for the `Html\Tag` component?

**Comment by niden** [comment]
And we will need some tests :) If you cannot just let me know

**Comment by ekmst** [comment]
> And we will need some tests :) If you cannot just let me know

@niden fixed :)

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
@ekmst Thank you!

File: CHANGELOG-4.0.md [hunk 1]
```diff
@@ -119,9 +119,11 @@
...
+- Changed the `Phalcon\Session` namespace by refactoring the component. `Phalcon\Session\Manager` is now the single component offering session manipulation by using adapters. Each adapter implements PHP's `SessionHandlerInterface`. Available adapters are `Phalcon\Session\Files`, `Phalcon\Session\Libmemcached`, `Phalcon\Session\Noop` and `Phalcon\Session\Redis`.  [#12921](https://github.com/phalcon/cphalcon/issues/12833), (https://github.com/phalcon/cphalcon/issues/11341), (https://github.com/phalcon/cphalcon/issues/13535)
 - Fixed `Phalcon\Mvc\Models` magic method (setter) is fixed for arrays  [#13661](https://github.com/phalcon/cphalcon/issues/13661)
 - Fixed `Phalcon\Mvc\Model::skipAttributes` and `Phalcon\Mvc\Model::allowEmptyColumns` allowEmptyStrings & skipAttributes repsect the column mapping. [#12975](https://github.com/phalcon/cphalcon/issues/12975), [#13477](https://github.com/phalcon/cphalcon/issues/13477) 
+- Changed the `Phalcon\Tag::renderTitle()` parameters such as `Phalcon\Tag::getTitle()`
+- Changed the `Phalcon\Html\Tag::renderTitle()` parameters such as `Phalcon\Html\Tag::getTitle()`
```
Comment (sergeyklay) [inline CHANGELOG-4.0.md]: @niden Can you please move these lines to another section, e.g. 

```
# [4.0.0-alpha2](https://github.com/phalcon/cphalcon/releases/tag/v4.0.0-alpha2) (2019-XX-XX)
## Changed
- Changed the `Phalcon\Tag::renderTitle()` parameters such as `Phalcon\Tag::getTitle()
- Changed the `Phalcon\Html\Tag::renderTitle()` parameters such as `Phalcon\Html\Tag::getTitle()`
```

**Pull request merged**